src/_variables.scss

```diff
@@ -20,3 +20,14 @@ $white: #F9F4F0;
   margin-left: 50%;
   transform: translateX(-50%)
 }
+
+@mixin button() {
```

Comment:
This button is going to be great to use across all our buttons! I like the consistency it will create.